## Hypothesis Testing

Hypothesis Statement:
H0:Null Hypothesis
H1:Alternate Hypothesis

Null Hypothesis (H0): The mean weekly operating cost is equal to the theoretical cost model, i.e.,(mu = 3050) where W =(1000 + 5*600).
Alternative Hypothesis (H1): The mean weekly operating cost is greater than the theoretical cost model, i.e.,(mu > 3050).

#Calculating Test Statistic

t=(X'-mu)/(sigma/sqrt(n))

In [25]:
import numpy as np
from scipy import stats

# Given data
sample_mean = 3050  # sample mean weekly cost
theoretical_mean = 1000 + 5 * 600  # theoretical mean weekly cost
sigma = 5 * 25  # standard deviation
n = 25  # sample size

# Calculate the test statistic
t_statistic = (sample_mean - theoretical_mean) / (sigma / np.sqrt(n))

# Critical value for one-tailed test at alpha = 0.05
alpha = 0.05
critical_value = stats.norm.ppf(1 - alpha)

# Decision
decision = "Reject H0" if t_statistic < critical_value else "Fail to reject H0"

# Conclusion
conclusion = "There is strong evidence to support the restaurant owners' claim." if decision == "Reject H0" else "There is not enough evidence to support the restaurant owners' claim."

t_statistic, critical_value, decision, conclusion

(-38.0,
 1.644853626951472,
 'Reject H0',
 "There is strong evidence to support the restaurant owners' claim.")

Conclusion:
Test Statistic: -38.0,
Critical Value: 1.644853626951472.

If the test statistic is greater than the critical value, we reject the null hypothesis.